scrapy/czlscrape/utils.py

```diff
@@ -0,0 +1,15 @@
+import re
+
+
+def guess_publication_type(text, rules):
+    text = text.lower()
+    text = re.sub(r'[șş]', 's', text)
+    text = re.sub(r'[țţ]', 't', text)
+    text = re.sub(r'[ăâ]', 'a', text)
+    text = re.sub(r'[î]', 'i', text)
+
+    for substr, publication_type in rules:
```

Comment:
Si totusi cred ca regulile de detectie a tipului unui proiect legislativ nu difera. Poate pentru comunicatii regulile sunt mai stufoase, iar pentru dialog sunt mai simple. Chiar si asa, ma astept ca o singura bucata de cod sa le acopere pe toate. Va scriu asta si cu celelalte scrapere in mind unde am vazut logica similara sau chiar identica.